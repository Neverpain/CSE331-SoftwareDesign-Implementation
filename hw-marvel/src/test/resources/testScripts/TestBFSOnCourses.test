# Tests the makeGraph method on the courses file to see that it is
# built correctly, prints out the paths of several nodes

LoadGraph g courses.csv

FindPath g cse142 cse143

FindPath g math124 math124

FindPath g cse142 math124

FindPath g eng101 cse143

FindPath g cse143 eng101
