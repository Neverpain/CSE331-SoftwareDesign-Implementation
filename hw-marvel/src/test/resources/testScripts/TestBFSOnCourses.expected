# Tests the makeGraph method on the courses file to see that it is
# built correctly, prints out the paths of several nodes

loaded graph g

path from cse142 to cse143:
cse142 to cse143 via cse

path from math124 to math124:

path from cse142 to math124:
no path found

path from eng101 to cse143:
no path found

path from cse143 to eng101:
no path found